src/notation/notationtypes.h

```diff
@@ -429,6 +429,14 @@ struct StaffConfig
     ClefTypeList clefTypeList;
     engraving::StaffType staffType;
 
+    StaffConfig(const engraving::Color& defaultColor)
```

Comment:
Hm, actually, sorry, that probably won't work. We can leave it as it is in this PR, but I do think we should refactor this later (perhaps it would be solved by making `EngravingConfiguration` global). Because I do think it's a bit ugly this way.